src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -678,10 +678,10 @@ public void CZ_DIALOG_SELECT(ChannelConnection conn, Packet packet)
 		[PacketHandler(Op.CZ_DIALOG_ACK)]
 		public void CZ_DIALOG_ACK(ChannelConnection conn, Packet packet)
 		{
-			var unkInt = packet.GetInt();
+			var type = packet.GetInt(); // 1 - Close from dialog, 0 - Close with escape
 
-			// Check state
-			if (conn.ScriptState.CurrentNpc == null)
+            // Check state
```

Comment:
Please set your IDE to use tabs, as outlined in our [coding conventions](https://github.com/aura-project/melia#coding-conventions).